src/docs/implementations/otlp.adoc

```diff
@@ -0,0 +1,131 @@
+
+[source, yaml]
+----
+management:
+  otlp:
+    metrics:
+      export:
+        # Supported configs
+        url: "https://otlp.example.com:4318/v1/metrics"
+        aggregationTemporality: "cumulative"
+        headers: "header1=value1"
+        resourceAttributes: "key=value"
+----
+
+1. url - The URL to which data will be reported. Defaults to `http://localhost:4318/v1/metrics`
+2. aggregationTemporality - https://opentelemetry.io/docs/specs/otel/metrics/data-model/#temporality[Aggregation temporality] determines how the additive quantities are expressed, in relation to time. The supported values are `cumulative` or `delta`. Defaults to `cumulative`. +
+*Note*: This config is introduced in version 1.11.0.
+3. headers - Additional headers to send with exported metrics. This can be used for authorization headers. By default, headers will be loaded from the config. If that is not set, they will be taken from the environment variables `OTEL_EXPORTER_OTLP_HEADERS` and `OTEL_EXPORTER_OTLP_METRICS_HEADERS`. If a header is set in both the environmental variables, the header in the latter will overwrite the former.
+4. resourceAttributes - https://opentelemetry.io/docs/specs/otel/resource/semantic_conventions/#service[Resource attributes] that will be used for all metrics published. By default micrometer adds the following resources,
+[%autowidth]
+|===
+|Key | Default value
+
+|telemetry.sdk.name
+|io.micrometer
+
+|telemetry.sdk.language
+|java
+
+|telemetry.sdk.version
+|<micrometer-core-version> (E.g: 1.11.0)
+
+|service.name
+|unknown_service
+|===
+
+If this config is empty, then resource attributes will be loaded from the environmental variable `OTEL_RESOURCE_ATTRIBUTES`. `service.name` can be overridden by the environmental variable `OTEL_SERVICE_NAME` and this takes precedence over other configs.
+
+== Supported metrics
+https://opentelemetry.io/docs/specs/otel/metrics/data-model/#metric-points[Metric points] define the different data-points that are supported in OTLP. Micrometer supports exporting the below data-points in OTLP format,
+
+1. https://opentelemetry.io/docs/specs/otel/metrics/data-model/#sums[Sums]
+2. https://opentelemetry.io/docs/specs/otel/metrics/data-model/#gauge[Gauge]
+3. https://opentelemetry.io/docs/specs/otel/metrics/data-model/#histogram[Histogram]
+4. https://opentelemetry.io/docs/specs/otel/metrics/data-model/#summary-legacy[Summary]
+
+The below table maps OTLP datapoint and the micrometer meters,
+[%autowidth]
+|===
+|OTLP DataPoint | Micrometer meter type
+
+|Counter, FunctionCounter
+| Sums
+
+|Gauge
+|Gauge
+
+|Histogram
+|Timer, DistributionSummary, LongTaskTimer, FunctionTimer (only sum and count are set)
+
+|Summary
+|Timer, DistributionSummary, LongTaskTimer
+|===
+
+*Note*: `max` on Histogram datapoint is only supported in Delta Aggregation Temporality.
```

Comment:
Could you please add a sentence why it is not supported on cumulative?
(Like OTLP defines it as max from app startup which is not very useful and Micrometer has a time window max).

Reply:
Added.